test/data/manifests/fedora-coreos-container-s390x.mpp.yaml

```diff
@@ -0,0 +1,177 @@
+                  tag: stable
+      - type: org.osbuild.ostree.selinux
+        options:
+          deployment:
+            osname: fedora-coreos
+            ref: ostree/1/1/0
+      - type: org.osbuild.zipl
+  - name: image
+    build: name:build
+    stages:
+      - type: org.osbuild.truncate
+        options:
+          filename: disk.img
+          size:
+            mpp-format-string: '{image.size}'
+      - type: org.osbuild.sfdisk
+        devices:
+          device:
+            type: org.osbuild.loopback
+            options:
+              filename: disk.img
+        options:
+          mpp-format-json: '{image.layout}'
+      - type: org.osbuild.mkfs.ext4
+        devices:
+          device:
+            type: org.osbuild.loopback
+            options:
+              filename: disk.img
+              start:
+                mpp-format-int: '{image.layout[''boot''].start}'
+              size:
+                mpp-format-int: '{image.layout[''boot''].size}'
+              lock: true
+        options:
+          uuid: 96d15588-3596-4b3c-adca-a2ff7279ea63
+          label: boot
+      - type: org.osbuild.mkfs.xfs
+        devices:
+          device:
+            type: org.osbuild.loopback
+            options:
+              filename: disk.img
+              start:
+                mpp-format-int: '{image.layout[''root''].start}'
+              size:
+                mpp-format-int: '{image.layout[''root''].size}'
+              lock: true
+        options:
+          uuid: 910678ff-f77e-4a7d-8d53-86f2ac47a823
+          label: root
+      - type: org.osbuild.copy
+        inputs:
+          tree:
+            type: org.osbuild.tree
+            origin: org.osbuild.pipeline
+            references:
+              - name:image-tree
+        options:
+          paths:
+            - from: input://tree/
+              to: mount://root/
+        devices:
+          boot:
+            type: org.osbuild.loopback
+            options:
+              filename: disk.img
+              start:
+                mpp-format-int: '{image.layout[''boot''].start}'
+              size:
+                mpp-format-int: '{image.layout[''boot''].size}'
+          root:
+            type: org.osbuild.loopback
+            options:
+              filename: disk.img
+              start:
+                mpp-format-int: '{image.layout[''root''].start}'
+              size:
+                mpp-format-int: '{image.layout[''root''].size}'
+        mounts:
+          - name: root
+            type: org.osbuild.xfs
+            source: root
+            target: /
+          - name: boot
+            type: org.osbuild.ext4
+            source: boot
+            target: /boot
+      - type: org.osbuild.zipl.inst
+        options:
+          kernel: "1"
+          location:
+            mpp-format-int: '0'
+        devices:
+          disk:
+            type: org.osbuild.loopback
+            options:
+              filename: disk.img
```

Comment:
you can go ahead and have the root and boot filesystems mounted if you add something like this (rebase on top of latest `main` branch to make sure you have all the recent enablement code):

```suggestion
      - type: org.osbuild.zipl.inst
        options:
          kernel: "1"
          location:
            mpp-format-int: '0'
        devices:                                                                                                                                                                             
          disk:                                                                                                                                                                              
            type: org.osbuild.loopback                                                                                                                                                       
            options:                                                                                                                                                                         
              filename: disk.img                                                                                                                                                             
              partscan: true                                                                                                                                                                 
        mounts:                                                                                                                                                                              
          - name: root                                                                                                                                                                       
            type: org.osbuild.xfs                                                                                                                                                            
            source: disk                                                                                                                                                                     
            partition:                                                                                                                                                                       
              mpp-format-int: '{image.layout[''root''].partnum}'                                                                                                                             
            target: /                                                                                                                                                                        
          - name: boot                                                                                                                                                                       
            type: org.osbuild.ext4                                                                                                                                                           
            source: disk                                                                                                                                                                     
            partition:                                                                                                                                                                       
              mpp-format-int: '{image.layout[''boot''].partnum}'                                                                                                                             
            target: /boot                                                                                                                                                                    
```

Reply:
No, i had this before. In short - zipl won't work with 3 loopdevs

Reply:
Ahh, i was using `start` & `size` to define mountpoints. Will rebase and check this approach

Reply:
yeah, with this strategy there is only 1 loop device.